### PR TITLE
#1 Gateway Adjustments and Documentation from Develop to main

### LINKED ISSUE #1: Create and adjust API gateway
**Acceptance Criteria:**

- API gateway is operational and successfully routes requests to microservices.
- Security measures are in place and functioning as intended.
- Monitoring tools provide insights into API usage and performance.
- Documentation is complete and accessible to the development team.

### PR DESCRIPTION
**Pull request by Nazaziik**

### CODE REVIEW

**Review by kossvat — DISMISSED** [review]
Looks good 👍

**Comment by VitalikMatvieiev** [comment]
> Looks good 👍

@kossvat how did you review it so fast?

File: Edu-platform_Gateway/Program.cs [hunk 1]
```diff
@@ -1,6 +1,16 @@
+using Ocelot.DependencyInjection;
+using Ocelot.Middleware;
+
 var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration.SetBasePath(builder.Environment.ContentRootPath)
```
Comment (VitalikMatvieiev) [inline Edu-platform_Gateway/Program.cs:6]: Do we need env versions?
I think so

File: Edu-platform_Gateway/ocelot.json [hunk 2]
```diff
@@ -0,0 +1,207 @@
+﻿{
```
Comment (VitalikMatvieiev) [inline Edu-platform_Gateway/ocelot.json:1]: Where did you get all of those endpoints?

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+# Change directory
+cd your-project
+
+# Restore dependencies
+dotnet restore
+
+# Build the project
+dotnet build
+
+# Run the project
```
Comment (VitalikMatvieiev) [inline README.md:50]: Is it any other steps to run it from mac? Should I install additional packages to my machine?

File: Edu-platform_Gateway/Program.cs (hunk 1 above)
Comment (VitalikMatvieiev) [reply Edu-platform_Gateway/Program.cs:6]: @Nazaziik

File: Edu-platform_Gateway/ocelot.json (hunk 2 above)
Comment (VitalikMatvieiev) [reply Edu-platform_Gateway/ocelot.json:1]: @Nazaziik

File: README.md (hunk 3 above)
Comment (VitalikMatvieiev) [reply README.md:50]: @Nazaziik

File: Edu-platform_Gateway/ApiGateway.csproj [hunk 4]
```diff
@@ -7,4 +7,8 @@
     <RootNamespace>Edu_platform_Gateway</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (VitalikMatvieiev) [inline Edu-platform_Gateway/ApiGateway.csproj:10]: Just test comments @Nazaziik @kossvat

**Review by VitalikMatvieiev — APPROVED** [review]
Congratulations !!! First PR DONE
istockphoto-1199025903-612x612

**Pull request closed without merging**